debezium-server/debezium-server-http/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,12 @@
+# Direct log messages to stdout
```

Comment:
Could you update this to logback config? We've been migrating off of log4j in a concurrent PR which just got merged. You can find examples for config and the dep in other modules.